src/Compilers/CSharp/Test/Semantic/Semantics/NativeIntegerTests.cs

```diff
@@ -1932,6 +1932,27 @@ static void verify(CSharpCompilation comp)
             }
         }
 
+        [WorkItem(42975, "https://github.com/dotnet/roslyn/issues/42975")]
+        [Fact]
+        public void AliasName_02()
+        {
+            var source =
+@"using N = nint;
```

Comment:
Why does this allowed, but `using I = int;` isn't? Seems inconsistent.

Reply:
Agreed. [https://github.com/dotnet/roslyn/issues/42975](https://github.com/dotnet/roslyn/issues/42975)

---
In reply to: [401738499](https://github.com/dotnet/roslyn/pull/42976#discussion_r401738499) [](ancestors = 401738499)